src/Microsoft.Identity.Web/DownstreamWebApiSupport/DownstreamWebApi.cs

```diff
@@ -113,11 +110,7 @@ public class DownstreamWebApi : IDownstreamWebApi
                 user,
                 new StringContent(JsonSerializer.Serialize(input), Encoding.UTF8, "application/json")).ConfigureAwait(false);
 
-            try
-            {
-                response.EnsureSuccessStatusCode();
```

Comment:
Can't response.EnsureSuccessStatusCode throw?
it seems it does: 
https://stackoverflow.com/questions/21097730/usage-of-ensuresuccessstatuscode-and-handling-of-httprequestexception-it-throws

Reply:
I reverted back to what we have for now. will take a more indepth look at this later